src/ImageSharp/Colors/Color.cs

```diff
@@ -112,6 +112,9 @@ public Color(uint packed)
             this.packedValue = packed;
         }
 
+        /// <inheritdoc />
+        public BulkPixelOperations<Color> BulkOperations => new BulkPixelOperations<Color>();
```

Comment:
Yeah it would better to allocate only one instance. 
BulkPixelOperations<T> is stateless + an instance is rarely requested so allocating it multiple times did not seemed to be a big issue for me.